specs/latest/1.0/index.html

```diff
@@ -3481,6 +3481,21 @@ <h2 class="no-toc">Table of contents</h2>
 
 </p>
 
+    <h3>No Shader Binaries</h3>
+
+<p>
+
+Accessing binary representations of compiled shaders is not supported in the WebGL API. This
+includes the OpenGL ES 2.0 ShaderBinary entry point. In addition, querying shader binary formats
+and the availability of a shader compiler via getParameter is not supported in the WebGL API.
```

Comment:
Sorry. @zhenyao, the pull request has been updated accordingly.  Please take another look.